src/main/webapp/styleNzinga.css

```diff
@@ -0,0 +1,245 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#location-container {
+  margin-top: 20px;
+}
+
+body,
+html {
+  height: 100%;
+}
+
+div {
+  margin: 0;
+}
+
+/* Add a black background color to the top navigation bar */
+.topnav {
+  overflow: hidden;
+  background-color: #e9e9e9;
+  text-align: center;
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order): Alphabetize declarations.

Here and elsewhere

Reply:
I'll fix in a later pr